### DISCUSSION TITLE
GitHub Actions workflow to run

### DISCUSSION THREAD

**Discussion by swchips** [community/community#136911] [Actions] [answered]
### Select Topic Area

Question

### Body

How can I configure a GitHub Actions workflow to run only on a specific branch?

**Comment by Haruniv** [comment, marked as answer]
To trigger a workflow only on a specific branch, you can use the on option in your .yml file. For example, to run the workflow only when a push is made to the main branch, you can configure it like this


e.g.yml file.

on:
  push:
    branches:
      - main

This will ensure that the workflow runs only when there are pushes to the main branch.
